cli/package.json

```diff
@@ -4,71 +4,43 @@
   "description": "A set of tools for interacting with the Common Architecture Language Model (CALM)",
   "main": "dist/index.js",
   "files": [
-    "dist"
+    "dist/"
   ],
   "scripts": {
-    "build": "tsc && npm run copy-calm-schema && npm run copy-spectral-rules",
-    "test": "jest --verbose",
+    "build": "tsc",
+    "watch": "tsc -watch -p ./",
+    "clean": "rimraf node_modules dist",
```

Comment:
What does this do?

Reply:
Just gives you the ability to go back to how the folder looked post-clone, `rimraf` is just a fancy wrapper around `rm -rf` :)

Reply:
It's honestly not required, it's a personal preference of mine to sometimes go back to how the code-base looks at clone to ensure that I don't have anything local holding things together.